packages/modules/providers/fulfillment-manual/src/services/manual-fulfillment.ts

```diff
@@ -34,7 +34,6 @@ export class ManualFulfillmentService extends AbstractFulfillmentProviderService
   }
 
   async calculatePrice(
```

Comment:
suggestion: shouldn't we use `CalculateShippingOptionPriceDTO` for the `data` and `context` for clearer typing?

Reply:
Good point, there is actually a TODO on top of the file, would prefer to revisit this in a separate PR to fix the entire file.